### DISCUSSION TITLE
Woocommerce version 9.3.2 problems on checkout

### DISCUSSION THREAD

**Discussion by Evelynukelele** [woocommerce/woocommerce#51619] [Checkout Flow]
Hi!

I've been updating Woocommerce, but it gives me an error:

'Er is een probleem opgetreden met de opgegeven factuuradres: is vereist'

So this means there's a problem with the invoice adress. Both my client as myself in testing mode got this message. I've tested everything; did I miss a box? Did I write everything right?

By the way; the way the checkout page looks, is stunning!

But anyway, nothing worked and you can't get into paying mode. 

So i backrolled the woocommerce plugin to version 9.2.3 and now everything works fine again. 
I'm a bit concerned though, because you said the update of Woocommerce is an important one, because of security?

I hope you can fix this bug. 

In the attachment you can see the error. 
<img width="1512" alt="Scherm­afbeelding 2024-09-23 om 11 58 18" src="https://github.com/user-attachments/assets/0fe82cb5-896d-404e-a44b-ce8af6bd6d54">

**Comment by senadir** [comment]
Hey! Thank you for the important report! I cannot seem to reproduce this, I don't see anything on your website that might indicate that. Can you try with a guest session and with a full address?

**Comment by niagaku909** [comment]
![Gambar WhatsApp 2024-10-23 pukul 09 31 36_9079f037](https://github.com/user-attachments/assets/56b7171f-0c99-4c3f-a0db-494edfe948d5)

i have this problem
how i can fix it

**Comment by nielslange** [comment]
@niagaku909, I've just responded to your query in [this discussion](https://github.com/woocommerce/woocommerce/discussions/52249#discussioncomment-11276020). Please try to avoid posting the same question in multiple threads as it can complicate the process of resolving your initial issue. Thank you for understanding!
